src/shared/featuredCategories.js

```diff
@@ -0,0 +1,22 @@
+export const featuredCategories = [
+  {
+    picturePath:
+      "https://plus.unsplash.com/premium_photo-1676234917179-a7b1ca98c984?w=700&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NDF8fGZvb2R8ZW58MHx8MHx8fDA%3D",
```

Comment:
Are these static photos that will always be displayed? If so, I would save the picture as a file in your repo. What if the link changes? What if unsplash is down?